roles/neutron-common/templates/etc/neutron/dhcp_agent.ini

```diff
@@ -1,13 +1,30 @@
 # {{ ansible_managed }}
 
 [DEFAULT]
+debug = True
+enable_isolated_metadata = True
+
+state_path = /var/lib/neutron
```

Comment:
This needs to be a new variable, `neutron.state_path` similar to what we do for nova and glance.